spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/PredictionService.scala

```diff
@@ -121,10 +124,9 @@ object PredictionService {
 
   def apply[T: ClassTag](
     model: Module[T],
-    nInstances: Int = 10,
-    nBackendThreads: Int = 2
+    nInstances: Int = 10
```

Comment:
why default is 10? Set default Engine.coreNumber?

Reply:
I think 10 looks like a reasonable number. In my opinion, while serving online, Engine.init() is unnecessary.